Ginger/GingerCoreNET/Run/RunListenerLib/CenteralizedExecutionLogger/AccountReportExecutionLogger.cs

```diff
@@ -39,7 +39,7 @@ public AccountReportApiHandler AccountReportApiHandler
         {
             get
             {
-                if (mAccountReportApiHandler == null || !mAccountReportApiHandler.IsConfigurationChanged())
+                if (mAccountReportApiHandler == null || mAccountReportApiHandler.IsConfigurationChanged())
```

Comment:
please explain why this change was needed

Reply:
In order to instantiate the  mAccountReportApiHandler attribute, it either needs to be null or the End Point URL needs to be changed. in the IsConfigurationChanged we check if the 2 strings "EndPointURL" and "WorkSpace.Instance.Solution.LoggerConfigurations.CentralLoggerEndPointUrl"  are equal. If they are equal the function returns false else true. Therefore the not gate in this snippet was unnecessary and hence removed